app/code/Magento/InventoryIndexer/Test/Integration/Stock/GetIsSalable.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryIndexer\Test\Integration\Stock;
+
+use Magento\Framework\App\ResourceConnection;
+use Magento\InventoryIndexer\Indexer\IndexStructure;
+use Magento\InventoryIndexer\Model\StockIndexTableNameResolverInterface;
+
+/**
+ * Get is salable for product.
+ */
+class GetIsSalable
```

Comment:
It is not a good idea to introduce a new implementation just for the sake of testing.

Please involve salability in integration test the way it will be actually used by Magento.

Reply:
It is just a temporary solution to check that we have correct value for is_salable in our index table. Now we don't have any service to check it.